### ISSUE TITLE
Request for timeout

### ISSUE THREAD

**Issue by Yang-ChanYoung** [t4d/StalkPhish#37]
I am trying gathering Phishing kits using StalkPhish. It is working very well.
But I have an trouble and think that I report the trouble.
Timeout parameter of requests.get is not working sometimes.
One requests.get in StalkPhish is not ended more than hours oftenly.

Reference : https://stackoverflow.com/questions/53242211/python-requests-timeout-not-working-properly

**Comment by t4d** [comment]
You are right, and for now I don't know what to do. Time will tell maybe  :)

**Comment by Sambyah9595** [comment]
Saya tidak tau kalo itu bahaya saya ada pencet aja
